week08_pomdp/practice_pytorch.ipynb

```diff
@@ -6,17 +6,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "import sys\n",
+    "import gym\n",
     "if 'google.colab' in sys.modules:\n",
-    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/atari_util.py\n",
-    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/env_pool.py\n",
-    "\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/f1d7764b276cadb7365b1fdb6f6dd3fbd4e7bd8d/week08_pomdp/atari_util.py\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/f1d7764b276cadb7365b1fdb6f6dd3fbd4e7bd8d/week08_pomdp/env_pool.py\n",
+    "    # Setup the attari driver for video games\n",
+    "    !wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/setup_colab.sh -O- | bash\n",
+    "    !touch .setup_complete\n",
+    "    # Install packages dependencies\n",
+    "    os.system(f\"pip install gym[atari,accept-rom-license]=={gym.__version__}\")\n",
+    "    \n",
     "# If you are running on a server, launch xvfb to record game videos\n",
     "# Please make sure you have xvfb installed\n",
-    "import os\n",
     "if type(os.environ.get(\"DISPLAY\")) is not str or len(os.environ.get(\"DISPLAY\")) == 0:\n",
     "    !bash ../xvfb start\n",
-    "    os.environ['DISPLAY'] = ':1'"
+    "    os.environ['DISPLAY'] = ':1'\n",
+    "\n",
+    "# Restart the Runtime\n",
+    "print(\"Restarting the Runtime...\")\n",
+    "os.kill(os.getpid(), 9)"
```

Comment:
You are correct! We are modifying the package of `Gym` by adding more extensions, such as `[atari,accept-rom-license]` so that we have to restart the kernel. I have tested it without the kernel killing process, and it showed me an error in `gym` environment.